lib/bamboo/adapters/send_grid_adapter.ex

```diff
@@ -368,6 +369,18 @@ defmodule Bamboo.SendGridAdapter do
 
   defp put_click_tracking(body, _), do: body
 
+  defp put_subscription_tracking(body, %Email{private: %{subscription_tracking_enabled: enabled}}) do
+    tracking_settings =
+      body
+      |> Map.get(:tracking_settings, %{})
+      |> Map.put(:subscription_tracking, %{enable: enabled, enable_text: enabled})
+
+    body
+    |> Map.put(:tracking_settings, tracking_settings)
```

Comment:
nit: might as well not pipe this since its only a single call

Reply:
Updated!